packages/perseus/src/widgets/orderer/orderer.tsx

```diff
@@ -527,41 +526,31 @@ class Orderer
         let sumHeight = 0;
 
         if (isHorizontal) {
-            _.each(
-                list,
-                function (opt, i) {
-                    const card = ReactDOM.findDOMNode(
-                        // eslint-disable-next-line react/no-string-refs
-                        // @ts-expect-error - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                        this.refs["sortable" + i],
-                    );
-                    // @ts-expect-error - TS2769 - No overload matches this call. | TS2339 - Property 'outerWidth' does not exist on type 'JQueryStatic'.
-                    const outerWidth = $(card).outerWidth(true);
-                    if (midWidth > sumWidth + outerWidth / 2) {
-                        index += 1;
-                    }
-                    sumWidth += outerWidth;
-                },
-                this,
-            );
+            list.forEach((opt, i) => {
+                const card = ReactDOM.findDOMNode(
+                    // eslint-disable-next-line react/no-string-refs
+                    this.refs["sortable" + i],
+                );
+                // @ts-expect-error - TS2769 - No overload matches this call. | TS2339 - Property 'outerWidth' does not exist on type 'JQueryStatic'.
+                const outerWidth = $(card).outerWidth(true);
```

Comment:
It might be good to remove some of these `@ts-expect-error` when possible since they bury **all** problems. I think this could have been something like:

```suggestion
                const outerWidth = ($(card) as any).outerWidth(true);
```